docs/implementations/luci.md

```diff
@@ -0,0 +1,16 @@
+# LUCI Implementation
+LUCI implements the WDI for USB and will be compatible with Bluetooth in the future. Due to LUCI's design and where it sits within the wheelchair electronics, there are a number of limitations with this implementation.
```

Comment:
Propose replacing second sentence with: "Not all WDI functions are not currently supported by LUCI."